### ISSUE TITLE
Please keep supporting classic WordPress versions

### ISSUE THREAD

**Issue by wp-cms** [AdvancedCustomFields/acf#786]
Currently working on a non-blocky version of WordPress ([wp-cms](https://github.com/wp-cms/wp-cms)), which doesn't contain block-related functions. There also is ClassicPress.

This issue is just to gently ask for your support by conditionally checking if the plugin is running on a "blocky" version of WP.

That way the plugin will keep working fine on WP-CMS and ClassicPress.

This is a fundamental plugin for the WP ecosystem, please don't leave non-blocky people behind.

Thanks.

**Comment by viktorix** [comment]
+1

**Comment by castellanario** [comment]
+1

**Comment by xxsimoxx** [comment]
+1

**Comment by salamarkesa** [comment]
Yes, please!

**Comment by harmony-developer** [comment]
Exactly, hope it keeps working because I will never use the blocks version of WordPress. And I use ACF PRO for multiple clients, with a valid license. Thanks!

**Comment by ElisabettaCarrara** [comment]
+1 Please, keep it compatible! Thanks!

**Comment by JoeRathbun** [comment]
+1

**Comment by araeubig** [comment]
+1
